Gems/Atom/Feature/Common/Code/Include/Atom/Feature/Mesh/StreamBufferViewsBuilderInterface.h

```diff
@@ -24,12 +25,36 @@ namespace AZ
             AZ_RTTI(AZ::Render::ShaderStreamBufferViewsInterface, "{3A80C85C-DD3A-4A1D-B564-291EB463CD0B}");
             virtual ~ShaderStreamBufferViewsInterface() = default;
 
-            //! Returns the Shader-bindable RHI::BufferView for the Vertex Indices from a particular mesh. 
-            virtual const AZ::RHI::Ptr<AZ::RHI::BufferView>& GetIndexBufferView() const = 0;
+            //! Returns the Shader-bindable RHI::BufferView for the Vertex Indices from a particular mesh.
+            //! REMARK: If the BindlessSrg is enabled, which typically is, this buffer view is already bound
+            //! as a ByteAddressBuffer
+            virtual const AZ::RHI::Ptr<AZ::RHI::BufferView>& GetVertexIndicesBufferView() const = 0;
+            //! Returns the StreamBufferView from the Vertex Indices, which is useful to know the actual
+            //! byte offset, byte count, etc related to the sub mesh(Mesh Index). 
+            virtual const AZ::RHI::IndexBufferView& GetVertexIndicesIndexBufferView() const = 0;
+            //! A shortcut to get the Read Index from the BindlessSrg for the IndexBufferView.
+            //! Alternatively, you could get the same index by getting it directly from the AZ::RHI::BufferView
+            //! returned by GetIndexBufferView()
+            virtual uint32_t GetVertexIndicesBindlessReadIndex(int deviceIndex) const = 0;
```

Comment:
can you add the convenience function `AZStd::unordered_map<int, uint32_t> GetVertexIndicesBindlessReadIndex() const;` similar to `BufferView::GetBindlessReadIndex()`, without a device index?

Reply:
Done.